.npmrc

```diff
@@ -1 +1,2 @@
 package-lock=false
+@zakodium:registry=https://registry.npmjs.org/
```

Comment:
This shouldn't be necessary. `@zakodium/eslint-config` is deprecated and should be replaced with `eslint-config-zakodium`.